src/lib/Components/Lists/ShowItemRow.tsx

```diff
@@ -119,7 +121,7 @@ export class ShowItemRow extends React.Component<Props, State> {
               ) : (
                 <OpaqueImageView width={58} height={58} imageURL={imageURL} />
               )}
-              <Flex flexDirection="column" flexGrow="1" width="197">
+              <Flex flexDirection="column" flexGrow="1" width={isScreenNarrow ? width / 7 : 197}>
```

Comment:
Just trying to understand why we're hardcoding a width. Was flexbox not enough to dynamically resize the row?

Reply:
If I don't set a width then the Gallery Infor container overflows to cut off the Save/Saved container:

<img width="381" alt="screen shot 2019-03-08 at 9 10 27 am" src="https://user-images.githubusercontent.com/10385964/54033356-14add000-4182-11e9-8a68-62bca4b1ee15.png">